docs/source/user-guide/introduction.md

```diff
@@ -26,7 +26,7 @@ in-memory format.
 DataFusion supports both an SQL and a DataFrame API for building
 logical query plans as well as a query optimizer and execution engine
 capable of parallel execution against partitioned data sources (CSV
-and Parquet) using threads.
+and Parquet) using
```

Comment:
`using` is redundant?

Reply:
Maybe removing "using threads" and saying "multi-threaded parallel execution" instead of "parallel exeuction" would read better.

Reply:
Looks good